packages/permission-log-controller/src/permissions.ts

```diff
@@ -0,0 +1,399 @@
+import { type Json, JsonRpcRequestStruct } from '@metamask/utils';
+import deepFreeze from 'deep-freeze-strict';
+
+export const CaveatTypes = Object.freeze({
+  restrictReturnedAccounts: 'restrictReturnedAccounts' as const,
+});
+
+export const RestrictedMethods = Object.freeze({
+  eth_accounts: 'eth_accounts',
```

Comment:
I see we're omitting Snaps RPC methods. We may have to think about how best to include them here in the future, but I think this makes sense for now.